server/persistence/persistence_test.go

```diff
@@ -113,23 +129,11 @@ func testCreatePlayer(t *testing.T, db persistence.DB) {
 	assert.EqualError(t, err, persistence.ErrPlayerAlreadyExists.Error())
 
 	p2 := model.Player{
-		ID:   model.PlayerID(rand.String(50)),
-		Name: `player 2`,
-		Games: map[model.GameID]model.PlayerColor{
-			model.GameID(1825): model.Blue,
-			model.GameID(26):   model.Red,
-			model.GameID(33):   model.Green,
-			model.GameID(108):  model.Red,
-		},
+		ID:    model.PlayerID(rand.String(50)),
+		Name:  `player 2`,
+		Games: map[model.GameID]model.PlayerColor{},
 	}
 	p2Copy := p2
-	// Don't keep the same memory space for the games copy
-	p2Copy.Games = map[model.GameID]model.PlayerColor{
-		model.GameID(1825): model.Blue,
-		model.GameID(26):   model.Red,
-		model.GameID(33):   model.Green,
-		model.GameID(108):  model.Red,
-	}
```

Comment:
TODO we should add this player to all of these games and check that it still works

Reply:
let's do this